### PR TITLE
Accordion usage rationale, update docs to reflect 1.0.8 main nav

### LINKED ISSUE #587: Rationale: when to use accordion
We should add rationale on good use cases for accordions. A common pattern seen is using accordions to replace headings and paragraphs on content pages. We need to explain why this is probably not good practice.

### PR DESCRIPTION
**Pull request by KiriHoy**
Rationale for when to use accordions. https://designsystem.apps.y.cld.gov.au/components/accordion/rationale/ 
Fixes #587

### CODE REVIEW

File: content/components/accordion/rationale/usage.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+Before using an accordion, consider whether the benefits outweigh [the negative usability impacts](https://www.nngroup.com/articles/accordions-complex-content/):
+
+**Hiding content makes it harder for a user to scan a webpage.**  If content is hidden inside an accordion it can be difficult for a user to scan a whole web page for content relevant to them. Web browser's 'Find on page...' search functions do not detect content hidden by accordions, making it harder for users to use their existing tools and behaviours to locate text. 
+
+**Accordions increase cognitive load.**  Forcing a user to click on each accordion in order to receive the full text fragments their user experience, causing them to switch focus between accordions in order to gather all the information. It is also possible that with hidden content, a user might ignore, or not see imortant information.
```
Comment (sukhrajghuman) [inline content/components/accordion/rationale/usage.md]:
```suggestion
**Accordions increase cognitive load.**  Forcing a user to click on each accordion in order to receive the full text fragments their user experience, causing them to switch focus between accordions in order to gather all the information. It is also possible that with hidden content, a user might ignore, or not see important information.  
```

File: content/components/accordion/rationale/usage.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+layout: component/rationale
+heading: When to use accordions
+---
+
+We recommend that accordions be used sparingly for primary content on a page.  While they can be appropriate for organising small, specific sections of content, accordions are not a suitable replacement for plain text that is well formatted. If a user requires all, or most of the information on the page it should be visible, not hidden inside an accordion.
```
Comment (gordongrace) [inline content/components/accordion/rationale/usage.md]: Either: "...plain text that is well-formatted" Or: "...well formatted plain text."

File: content/components/accordion/rationale/usage.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
+layout: component/rationale
+heading: When to use accordions
+---
+
+We recommend that accordions be used sparingly for primary content on a page.  While they can be appropriate for organising small, specific sections of content, accordions are not a suitable replacement for plain text that is well formatted. If a user requires all, or most of the information on the page it should be visible, not hidden inside an accordion. 
+
+Before using an accordion, consider whether the benefits outweigh [the negative usability impacts](https://www.nngroup.com/articles/accordions-complex-content/):
+
+**Hiding content makes it harder for a user to scan a webpage.**  If content is hidden inside an accordion it can be difficult for a user to scan a whole web page for content relevant to them. Web browser's 'Find on page...' search functions do not detect content hidden by accordions, making it harder for users to use their existing tools and behaviours to locate text.
```
Comment (gordongrace) [inline content/components/accordion/rationale/usage.md]: Web browsers'

File: content/components/accordion/rationale/usage.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+Before using an accordion, consider whether the benefits outweigh [the negative usability impacts](https://www.nngroup.com/articles/accordions-complex-content/):
+
+**Hiding content makes it harder for a user to scan a webpage.**  If content is hidden inside an accordion it can be difficult for a user to scan a whole web page for content relevant to them. Web browser's 'Find on page...' search functions do not detect content hidden by accordions, making it harder for users to use their existing tools and behaviours to locate text. 
+
+**Accordions increase cognitive load.**  Forcing a user to click on each accordion in order to receive the full text fragments their user experience, causing them to switch focus between accordions in order to gather all the information. It is also possible that with hidden content, a user might ignore, or not see important information.
```
Comment (gordongrace) [inline content/components/accordion/rationale/usage.md]: "...each accordion to receive..."

**Pull request merged**